lwtnn.spec

```diff
@@ -1,17 +1,32 @@
-### RPM external lwtnn 1.0
+### RPM external lwtnn 2.4
+
 Source: https://github.com/lwtnn/lwtnn/archive/v%{realversion}.tar.gz
-BuildRequires: py2-pkgconfig
+BuildRequires: py2-pkgconfig ninja cmake
```

Comment:
Remove `py2-pkgconfig` as it's not used anymore. It was used for old makefiles: `makefile:CXXFLAGS     += $(shell pkg-config eigen3 --cflags)`